python/cudf/dataframe/series.py

```diff
@@ -1479,6 +1481,33 @@ def digitize(self, bins, right=False):
 
         return Series(numerical.digitize(self._column, bins, right))
 
+    def shift(self, periods=1, freq=None, axis=0, fill_value=None):
+        """Shift values of an input array by periods positions and store the
+        output in a new array.
+
+        Notes
+        -----
+        Shift currently only supports float and integer dtype columns with
+        no null values.
+        """
+        assert axis in (None, 0) and freq is None and fill_value is None
+
+        if self.null_count != 0:
+            raise AssertionError("Diff currently requires columns with no "
```

Comment:
Should this say "Shift currently requires columns with no null values"?

Reply:
Yes, it should. Good catch. I should've caught that beforehand.